.changeset/yellow-forks-knock.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-cost-insights': patch
+'@backstage/test-utils': patch
```

Comment:
This should be split up into a separate changeset as mentions of cost insights does not need to go into the changelog for test-utils 😄

Reply:
Oh ok, updated 😄